### ISSUE TITLE
[BUG] Plotting errors out at 39% Phase 2/4 

### ISSUE THREAD

**Issue by Yarnila** [Chia-Network/chia-blockchain#1633]
I have a 4TB M2 as my Temp Drive so i can dual plot K32s (4). 
Yesterday i went and tried 6 they errored out at 39% without an error showing from the 6 plots 1 was at 88% and just stopped without an error. I wiped the temp drive after shutting down chia and restarted with 4 k32s to test. Now i at least am getting an error this time, but have no clue as to why. Ram is good, Threads are good and space is enough. 

I have the following settings: 
Ram is set to Standard
Threads is set to 16 (4) per Plot 

The rest is as is. 

This is the error i am getting: "Only read 0 of 4194304 bytes at offset 486539104 from I:\\plot-k32-2021-04-01-08-12-HASHNR plot table 6 tmp with length 49284775110 Error 1:Retrying in 5 minutes 

- OS: windows 10
- CPU: AMD Ryzen 9 3950X 
- RAM: 32GB 3200 Ripjaws

**Comment by chiachain** [comment]
Same here. Doing 2 in parallel - sometimes works, sometimes this same error.

**Comment by hoffmang9** [comment]
These are hardware errors with either your disks or RAM.

**Comment by Yarnila** [comment]
The Drive is Brand new and it worked fine before I switched out to the bigger drive. Had no issues with the ram sofar either that's all good. This machine aint even that old.

**Comment by Yarnila** [comment]
Is it because i went with 4 plots right away in parallel do I have to choose 2 and then the next 2 and so on?

**Comment by hoffmang9** [comment]
4 plots are going to need 1425 GB if they all hit the same stage at the same time.

**Comment by Yarnila** [comment]
Yeah, that's fine what I mean is I set up 4 plots from the menu in parallel what I am wondering is should I have done 2 parallel plots gave them a temp folder then the next 2 a different temp folder cos it looks like they were trying to access the same k32 tmp file . Anyway, I have created 3 temp folders and selected 2 plots each time. I have a 4TB Temp drive so that should not even be an issue. I will let you know if these 5 plots go through this time.
